tracer/build/PackageVersionsGeneratorDefinitions.json

```diff
@@ -235,7 +235,12 @@
       "5.*.*",
       "6.*.*",
       "7.*.*"
-    ]
+    ],
+    "VersionConditions": [{
+      "MinVersion": "3.1.32",
+      "MaxVersionExclusive": "4.0.0",
+      "ExcludeTargetFrameworks": ["netcoreapp3.0"]
+    }]
```

Comment:
This appears to consistently seg-fault on alpine only, on .NET Core 3.0 only, hence the exclusion for now to unlock CI